app/controllers/api/users_controller.rb

```diff
@@ -26,5 +26,16 @@ def login
         }
       }.to_json
     end
+
+    def show
+      user = User.find(params[:id])
+      render json: {
```

Comment:
Let's make our code look cleaner. One way to do this is by adding new lines between sections of the code. Add an empty line between finding the user and rendering the result.